packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -82,12 +82,15 @@ function UncontrolledInnerBlocks( props ) {
 		[ clientId ]
 	);
 
+	// Avoid creating new objects if there is no layout defined.
+	const emptyLayout = {};
+
```

Comment:
We should move this outside the component to avoid re-creating variables on every re-render. You can find similar instances of `EMPTY_OBJECT` in the codebase.

Reply:
Oh, got it, thanks!

Reply:
Done!